docs/openapi.yaml

```diff
@@ -1209,26 +1209,6 @@ components:
           example: "5001419121"
         type:
           $ref: "#/components/schemas/ChargeType"
-        failureReason:
```

Comment:
We are not going to remove this entirely. We will still use it to communicate if there was a technical reason for the failure of a charge, or if we know that the charge is going to fail (if charge amount is higher than customers max amount for instance)

Reply:
But we will nost likely not give any failureReason during retry period unless there is something the merchant can do.

Reply:
We should add an event log, and not have failure/problem info in the charge details itself. We need an array of events. We already have a state, so the event log will have the details.